sample-app/src/main/java/io/rover/app/sample/SampleMainActivity.kt

```diff
@@ -16,20 +16,41 @@ class SampleMainActivity : AppCompatActivity() {
              RoverActivity.makeIntent(this, experienceId = "my-experience-id", campaignId = null)
          )
```

Comment:
I think we have to rethink this bit of the example.  Right now it messes up the deep link handling example below, and does not work as-is.  Perhaps do it instead of `return` if intent.data is null.